### ISSUE TITLE
quick questions regarding your paper

### ISSUE THREAD

**Issue by jackswl** [microsoft/LatentSpatialMemory#16]
Hey! wonderful work again.

Just a few quick questions, hoping you can clarify for me!

1) For controlnet side branch, did you do {0, 4, 8, 12, 16, 20, 24, 28}? Or is it {0,2,4,...}?
2) Did you initialize from the wan 2.2 blocks? It is NOT random init, right?
3) Can you share your learning rate scheduler/training steps etc in specifics?

Thank you so much! Looking forward to your response @MD-Student
